doc/docs/master/reference/scaling_limits.md

```diff
@@ -0,0 +1,58 @@
+# Pipeline Scaling Limits in Community Edition
+
+Pachyderm Community Edition **limits the number of pipelines** users can deploy and **caps the parallelism** settings of those pipelines.  
+
+If your DAGs require more than what our Community Edition allows you to run, [ask for a free trial and receive a time-limited enterprise license token](https://www.pachyderm.com/trial). 
+
+## Scaling Limits
+
+|**Number of pipelines** deployed| **Number of workers** for each pipeline|
+|------|------|
+|Community Users can deploy **up to 16 pipelines**.| Community Users can run **up to 8 workers in parallel** on each pipeline.|
+
+Are you hitting those limits? You are likely to be needing our assistance in optimizing your DAGs. We are here to help! See [Lift the limitation](#lift-the-limitation) and request a free trial of our Enterprise Edition.
+
+## Lift the limitations
+
+To experiment with Pachyderm without limit, we provide an easy way to request an [**Enterprise Edition trial token**](https://www.pachyderm.com/trial). Fill in the form (link provided) and receive an enterprise license token.
+
+!!! Info "You might qualify for a free Enterprise license." 
+    Pachyderm offers activation keys for proofs-of-concept, startups, academic, nonprofit, or open-source projects. [Tell us about your project!](https://www.pachyderm.com/trial). 
+
+## What happens when you exceed those limits?
+
+As a general rule, Pachyderm provides a message in the STDERR whenever a limit is encountered.
+
+For example, each `pachctl` command invoking an enterprise feature (such as `pachctl auth,` `pachctl deploy ide`...) logs an alert message in your STDERR with a link to the [Enterprise Edition documentation](https://docs.pachyderm.com/latest/enterprise/) for more information. 
```

Comment:
The link is the trial link, not the enterprise link, for all errors.

Pachctl only emits the enterprise link on deploy.